sphinxcontrib/phpdomain.py

```diff
@@ -22,7 +22,7 @@
 from sphinx.util.nodes import make_refnode
 from sphinx.util.compat import Directive
 from sphinx.util.docfields import Field, GroupedField, TypedField
-
+from sphinx.domains.python import _pseudo_parse_arglist
```

Comment:
Given that this starts with a `_`, I'm not sure we should be using it as that prefix is generally used to indicate private or unstable functions.